tools/exe_hint_rerun.ps1

```
@@ -0,0 +1,62 @@
+$si.UseShellExecute = $false
+$si.RedirectStandardOutput = $true
+$si.RedirectStandardError = $true
+$p = New-Object System.Diagnostics.Process
+$p.StartInfo = $si
+$p.Start() | Out-Null
+
+$outTask = $p.StandardOutput.ReadToEndAsync()
+$errTask = $p.StandardError.ReadToEndAsync()
+$exited = $p.WaitForExit($killMs)
+if (-not $exited) {
+    try { $p.Kill() } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)